docs/index.bs

```diff
@@ -2544,14 +2544,14 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       : Input
       :: |job|, a [=job=]
-      :: |reason|, an [=exception=]
+      :: |errorName|, a string
       : Output
       :: none
 
-      1. If |job|'s [=job/client=] is not null, [=queue a task=], on |job|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |job|'s [=job/job promise=] with an exception that is a copy of |reason|, in |job|'s [=job/client=]'s [=environment settings object/Realm=].
+      1. If |job|'s [=job/client=] is not null, [=queue a task=], on |job|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |job|'s [=job/job promise=] with a [=exception/create|new=] [=exception=] with |errorName| and a user agent-defined [=exception/message=].
       1. For each |equivalentJob| in |job|'s [=list of equivalent jobs=]:
-          1. If |equivalentJob|'s [=job/client=] is null, continue to the next iteration of the loop:
-          1. [=Queue a task=], on |equivalentJob|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |equivalentJob|'s [=job/job promise=] with an exception that is a copy of |reason|, in |equivalentJob|'s [=job/client=]'s [=environment settings object/Realm=].
+          1. If |equivalentJob|'s [=job/client=] is null, [=iteration/continue=].
+          1. [=Queue a task=], on |equivalentJob|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |equivalentJob|'s [=job/job promise=] with a [=exception/create|new=] [=exception=] with |errorName| and a user agent-defined [=exception/message=].
```

Comment:
This structure doesn't quite work, because you need to say "DOMException" for the "SecurityError" cases, and not say DOMException for the TypeError case.

I'd maybe make the argument "_errorData_, the information necessary to [=exception/create|create an exception=]". Then you could say "with _job_ and `"SecurityError"` `DOMException`", or "with _job_ and `TypeError`".

Also, you should state the realm. Inside a queued task, you've "lost" the current realm, since current realm refers to that of the executing function object.